pkg/provider/okta/okta.go

```diff
@@ -811,7 +813,7 @@ func verifyMfa(oc *Client, oktaOrgHost string, loginDetails *creds.LoginDetails,
 	switch mfa := challengeContext.mfaIdentifer; mfa {
 	case IdentifierYubiMfa:
 		return gjson.Get(challengeContext.challengeResponseBody, "sessionToken").String(), nil
-	case IdentifierSmsMfa, IdentifierTotpMfa, IdentifierOktaTotpMfa, IdentifierSymantecTotpMfa:
+	case IdentifierSmsMfa, IdentifierEMailMfa, IdentifierTotpMfa, IdentifierOktaTotpMfa, IdentifierSymantecTotpMfa:
```

Comment:
Did you mean `IdentifierEmailMfa`? `IdentifierEMailMfa` looks odd?

Also, we can just add a test to exercise this case so that we know when it is hit.

Reply:
Yes, my bad,  there's some name casing issue, I'll update.